projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/features/assignments.ts

```diff
@@ -271,6 +271,8 @@ export function assignmentsTest(config: MyCompanyConfig) {
           .contains(ASSIGNMENT_LABELS.MANAGE)
           .click();
         cy.wait('@getAssignable');
+        // waiting for the UI element to finish loading. TODO: This is not ideal. there has to be a way to work around loading states.
+        cy.wait(1000);
```

Comment:
Agreed that waits should not be required.  On the other hand I am aware we are struggling with test stability/reliability.  If you think this improves things until we find a better solution, then it's ok since it's on the maintenance branch.

Reply:
For the life of me I can't recall how precisely this change fixed the test. The code looks quite different in current `develop`; I could copy that over, but I'm not sure if anything else had changed.

`develop` code as reference:
https://github.com/SAP/spartacus/blob/develop/projects/storefrontapp-e2e-cypress/cypress/helpers/b2b/my-company/features/assignments.ts